backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc

```diff
@@ -61,10 +61,21 @@ spec:
     ttl: 720h0m0s
 EOF
 ----
+
 <1> `cron` expression to schedule the backup, for example, `0 7 * * *` to perform a backup every day at 7:00.
++
+[NOTE]
+====
+To schedule a backup at specific intervals, enter the `<duration in minutes>` in the following format:
```

Comment:
Underscores needed for multi-word replaceable values ([repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values)):
```suggestion
To schedule a backup at specific intervals, enter the `<duration_in_minutes>` in the following format:
```